lib/common/common/Jobs.inl

```diff
@@ -52,7 +52,7 @@ namespace JsUtil
     template<class TJobManager, class TJobHolder>
     void JobProcessor::PrioritizeJobAndWait(TJobManager *const manager, const TJobHolder holder, void* function)
     {
-        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>;
+        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>();
```

Comment:
Oh I missed that there was no variable name. Does this rely only on side effects or does it rely on that value being kept around on the stack? If the latter, I wouldn't be surprised if calling the constructor but not providing a name for the variable might be undefined behavior. Maybe you should also add a name?